app/models/identified_client.rb

```diff
@@ -133,8 +136,8 @@ def download_data
         middle_name,
         last_name,
         date_of_birth,
-        Gender.find_by(numeric: gender)&.text,
-        Race.find_by(numeric: race)&.text,
+        gender_descriptions.join(', '),
+        race_descriptions.join(', '),
```

Comment:
This is for the identified clients CSV download. @eanders do you think this is okay, or should we have separate columns per value? I don't believe this gets imported anywhere.